dashboard/installation/dashboard-migration.md

```diff
@@ -32,6 +38,11 @@ The data migration process forms part of the release pipeline. Please refer to [
 
 In addition to these values, the SQL server password **must** be stored as a secret in your Invictus Azure Key Vault with the name `invictussqlserverpassword`.
 
+Make sure that the PowerShell script in the release pipeline which runs Deploy.ps1 has the following parameters:
+```
+    azurePowerShellVersion: LatestVersion
+    pwsh: false
+```
```

Comment:
This sub-section is dedicated to the SQL Data migration. Would be better to place this in a section above.